pyserini/vsearch/index.py

```diff
@@ -1,101 +0,0 @@
-#
```

Comment:
where did this go?

Reply:
In pyserini index nmslib already https://github.com/castorini/pyserini/blob/master/pyserini/index/nmslib.py